src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -155,6 +155,12 @@ uct_component_t uct_cuda_copy_component = {
         .table          = uct_cuda_copy_md_config_table,
         .size           = sizeof(uct_cuda_copy_md_config_t),
     },
+    .cm_config          = {
+        .name           = "",
+        .prefix         = "",
+        .table          = NULL,
+        .size           = 0,
+    },
```

Comment:
why even need to initialize cm_config for components without CM flag?

Reply:
all cm_configs are added to the ucx_config_global_list to print them later.
https://github.com/openucx/ucx/pull/4424/files#diff-31769de84b67b2f5ec210c6065cbedf6R158
can only the ones from sockcm and rdmacm be added?